autotests/CMakeLists.txt

```
@@ -26,4 +26,9 @@ if(${PROJECT_NAME}_ENABLE_E2EE)
     quotient_add_test(NAME testolmutility)
     quotient_add_test(NAME testfilecrypto)
     quotient_add_test(NAME testkeyverification)
+    if(${BUILD_SHARED_LIBS})
+        message("Cross-Signing test requires building libQuotient statically")
+    else()
+        quotient_add_test(NAME testcrosssigning)
+    endif()
```

That conditional should not be needed any more.
```suggestion
    quotient_add_test(NAME testcrosssigning)
```